src/components/MintContent.jsx

```diff
@@ -103,9 +104,17 @@ export default function MintContent() {
     console.log('balance',(getWalletInfo()['amount'] / 1000000).toFixed(3));
     setBalance((getWalletInfo()['amount'] / 1000000).toFixed(3));
   }, [])
+  
+  var sessionStorageSetHandler = function(e) {
```

Comment:
Is there any way to condense this code? Maybe you can write the sessionStorageSetHandler just once, export it and import it in the necessary files. If you pass "setLoadingText" as an argument, I think it would still work.